### PR TITLE
fix image plugin layout

### PR DESCRIPTION
**Pull request by stefanfoulis**

### CODE REVIEW

**Comment by lory87** [comment]
:+1:

File: cmsplugin_filer_image/cms_plugins.py [hunk 1]
```diff
@@ -22,14 +22,16 @@ class FilerImagePlugin(CMSPluginBase):
...
+                'image_url',
                 'alt_text',
             ]
         }),
         (_('Image resizing options'), {
             'fields': (
                 'use_original_image',
-                ('width', 'height', 'crop', 'upscale'),
+                ('width', 'height',),
+                ('crop', 'upscale'),
```
Comment (Chive) [inline cmsplugin_filer_image/cms_plugins.py]: consistency? :)

Comment (stefanfoulis) [inline cmsplugin_filer_image/cms_plugins.py]: jaja

Comment (stefanfoulis) [inline cmsplugin_filer_image/cms_plugins.py]: @Chive https://github.com/divio/cmsplugin-filer/commit/fb68bbc55ae3c0c4364fc7d3fedb8daa82e98026

Comment (Chive) [inline cmsplugin_filer_image/cms_plugins.py]: :tada:

**Pull request merged**